api/Window.json

```
@@ -4686,6 +4686,44 @@
           }
         }
       },
+      "pushManager": {
+        "__compat": {
+          "spec_url": "https://w3c.github.io/push-api/#dom-pushmanagerattribute-pushmanager",
+          "support": {
+            "chrome": {
+              "version_added": false,
+              "impl_url": "https://crbug.com/382298314"
+            },
+            "chrome_android": "mirror",
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false,
+              "impl_url": "https://bugzil.la/1935325"
+            },
+            "firefox_android": "mirror",
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": "18.5"
+            },
+            "safari_ios": {
+              "version_added": "18.4",
+              "notes": "Notifications are supported in web apps saved to the home screen."
+            },
```

- Implemented via https://github.com/WebKit/WebKit/commit/b96ecd4d82e2ac0f15580a7a1ea7e54b653d3867 in [WebKit 621.1.10](https://github.com/WebKit/WebKit/blob/b96ecd4d82e2ac0f15580a7a1ea7e54b653d3867/Configurations/Version.xcconfig#L24-L26) before Safari 18.4.
- The feature was already enabled before via https://github.com/WebKit/WebKit/commit/a786946331264a29f40aca673a1953d573fa60d7 in [WebKit 621.1.9](https://github.com/WebKit/WebKit/blob/a786946331264a29f40aca673a1953d573fa60d7/Configurations/Version.xcconfig#L24-L26).

Assuming you didn't test explicitly in Safari 18.4 on macOS, I guess this landed at the same time.

```suggestion
            "safari": {
              "version_added": "18.4"
            },
            "safari_ios": {
              "version_added": "18.4",
              "notes": "Notifications are supported in web apps saved to the home screen."
            },
```